source-requirements.txt

```diff
@@ -4,6 +4,7 @@ httplib2==0.9
 kafka-python==0.9.0-9bed11db98387c0d9e456528130b330631dc50af
 ntplib==0.3.2
 pg8000==1.9.6
+psycopg2==2.6
```

Comment:
This needs some compiled stuff, it should probably live in the source-optional-requirements file.